.github/workflows/icu_ci.yml

```diff
@@ -268,19 +270,34 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: ICU4C with clang
+      - name: Install doxygen
```

Comment:
FYI: Frank says that he split some large steps into smaller ones so that it's easier to figure out what fails (if something fails).